server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/http/DownloadRoutes.java

```diff
@@ -74,15 +74,15 @@
     private final BlobManager blobManager;
     private final SimpleTokenFactory simpleTokenFactory;
     private final MetricFactory metricFactory;
-    private final AuthenticationReactiveFilter authenticationReactiveFilter;
+    private final AuthenticationFilter authenticationFilter;
 
     @Inject
     @VisibleForTesting
-    DownloadRoutes(BlobManager blobManager, SimpleTokenFactory simpleTokenFactory, MetricFactory metricFactory, AuthenticationReactiveFilter authenticationReactiveFilter) {
+    DownloadRoutes(BlobManager blobManager, SimpleTokenFactory simpleTokenFactory, MetricFactory metricFactory, AuthenticationFilter authenticationFilter) {
         this.blobManager = blobManager;
         this.simpleTokenFactory = simpleTokenFactory;
         this.metricFactory = metricFactory;
-        this.authenticationReactiveFilter = authenticationReactiveFilter;
+        this.authenticationFilter = authenticationFilter;
```

Comment:
it's probably not a filter

Reply:
Better idea for a name?

AuthenticationPerformer would be ok for you?

Reply:
Authenticator?